sql/src/main/java/io/crate/executor/transport/TransportExecutor.java

```diff
@@ -193,17 +197,16 @@ public Void visitQueryAndFetch(QueryAndFetch plan, Job job) {
                     transportActionProvider.transportJobInitAction(),
                     transportActionProvider.transportCloseContextNodeAction(),
                     handlerSideDataCollectOperation,
+                    streamerVisitor,
                     circuitBreaker,
-                    plan.localMergeNode(),
```

Comment:
make localMergeNode in ExecutionNodesTaks Nullable ( start fails if mergeNode is null)